src/ripple/app/misc/ValidatorList.h

```diff
@@ -207,17 +215,18 @@ class ValidatorList
         Reset the trusted keys based on latest manifests, received validations,
         and lists.
 
-        @param seenValidators Set of public keys used to sign recently
-        received validations
+        @param seenValidators Set of NodeIDs of validators that have signed
+        recently received validations
+
+        @return TrustedKeyChanges instance with newly trusted or untrusted
+        master public keys.
 
         @par Thread Safety
 
         May be called concurrently
     */
-    template<class KeySet>
-    void
-    onConsensusStart (
-        KeySet const& seenValidators);
+    TrustChanges
```

Comment:
Do you think this warrants a name change to `onConsensusStart`?

Reply:
Good call; `updateTrustedKeys` matches the doxygen comment.

Reply:
Actually, that was already out of sync a bit based on these changes. `updateTrusted` is better.